package.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Not sure how much devmtn has changed and if they taught this, but you can start a brand new package.json by running `npm init` and it will walk you through setting up your config. I added a few helpful scripts as well.

Reply:
Now that i think of it, i would like to request a change from "tests suck and ..." to "TODO" because "tests suck" doesn't look good to a potential employer ;)